calibration/PARADK/CGINS-PARADK-20436__20150812.csv

```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20436,CC_dark_offset,1.5,
+20436,CC_scale_wet,9.92e-18,
```

Comment:
CC_scale_wet should be 1.03e-17 and CC_dark_offset 1.2

Reply:
fixed